pkg/api/types.go

```diff
@@ -123,8 +123,8 @@ type OrchestratorProfile struct {
 // KubernetesConfig contains the Kubernetes config structure, containing
 // Kubernetes specific configuration
 type KubernetesConfig struct {
-	KubernetesHyperkubeSpec string `json:"kubernetesHyperkubeSpec,omitempty"`
-	KubectlVersion          string `json:"kubectlVersion,omitempty"`
+	KubernetesImageBase string `json:"kubernetesImageBase,omitempty"`
+	KubectlVersion      string `json:"kubectlVersion,omitempty"`
```

Comment:
I will take this issue to extract version from hyperkube image url. Let me do it:)